lsp-pwsh.el

```diff
@@ -256,25 +260,24 @@ Must not nil.")
 (defun lsp-pwsh--extra-init-params ()
   "Return form describing parameters for language server.")
 
-(defun lsp-pwsh--apply-code-action-edits (action)
+(lsp-defun lsp-pwsh--apply-code-action-edits ((&pwsh:ApplyCodeActionEdits :command :arguments))
   "Handle ACTION for PowerShell.ApplyCodeActionEdits."
-  (-if-let* ((command (gethash "command" action))
-             ((&hash "StartLineNumber" "EndLineNumber"
-                     "StartColumnNumber" "EndColumnNumber" "Text")
-              (lsp-seq-first (gethash "arguments" action)))
+  (-if-let* (((&pwsh:ApplyCodeActionEditsArguments :start-line-number :end-line-number
+                                                   :start-column-number :end-column-number :text)
+              (lsp-seq-first arguments))
              (edits `[,(ht ("range" (ht ("start"
```

Comment:
ht usage should be fixed as well. There are 3 options:

1. Use `lsp-set-*` to update the structures inplace
2. Do the work on top of plists which has generated contractors as well
3. Me merging the lsp-protocol.el change from plist.